rancher2/data_source_rancher2_project.go

```diff
@@ -0,0 +1,115 @@
+package rancher2
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/rancher/norman/types"
+)
+
+func dataSourceRancher2Project() *schema.Resource {
+	return &schema.Resource{
+		Exists: dataSourceRancher2ProjectExists,
+		Read:   dataSourceRancher2ProjectRead,
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Description: "ID of the cluster to whom the project belongs",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"name": {
+				Description: "Name of the project",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"description": {
+				Description: "Description of the project",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"project_id": {
+				Description: "Cluster-wide unique identifier of the project",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
```

Comment:
I am not really sure. I *was* unsing `id` instead of `project_id` at first, but decided that it would be better to use the field `project_id`, because that's how the field is named in the `rancher2_project` resource after all.

I think consistency makes it easier for users in this case. What do you think?

Reply:
Agreed to maintain consistency, but the field is named `id` in `rancher2_project`, it's called `project_id` on resources that reference it, like `rancher2_namespaces`. In fact, it's the id of the resource (`porject_id` is duplicated info) then may be is more consistent to call it just `id`. :)

Not a big deal here, just mention it but it's not mandatory to remove it.

Reply:
Haha. Ok... You got me. I *am* convinced. Renaming it...